src/View/CrudView.php

```diff
@@ -6,6 +6,32 @@
 
 class CrudView extends View
 {
+    public $layout = 'CrudView.default';
+
+    public function initialize(array $config = [])
```

Comment:
I'm not extending BootstrapUI, I'm extending the core View class. So this is exactly how this should work.